Assets/_Project/Resources/ExampleData/waypoints.json

```diff
@@ -0,0 +1,59 @@
+﻿[
+  {
+    "waypoint_id": "gliwice.rynek.neptun",
+    "coordinates": [
+      50.29418,
+      18.66561
+    ],
+    "zone_name": "Rynek",
+    "type": "station",
+    "exercises": [
+      "2_quick_breaths"
+    ],
+    "displayed_name": "Rynek - Neptun",
+    "historical_facts": [
+      "gliwice.neptun"
+    ],
+    "navigation_audio": "audio.navigation.neptun"
+  },
+  {
+    "waypoint_id": "gliwice.rynek.ratusz",
+    "coordinates": [
+      50.29387,
+      18.66577
+    ],
+    "zone_name": "Rynek",
```

Comment:
Do we even Zones? The last I heard they are completely gone.

Reply:
I would leave them, I think they still might serve some purpose.